harness/regExpUtils.js

```diff
@@ -31,23 +31,65 @@ function buildString(args) {
   return result;
 }
 
-function testPropertyEscapes(regex, string, expression) {
-  if (!regex.test(string)) {
+function printCodePoint(codePoint) {
+  const hex = codePoint
+    .toString(16)
+    .toUpperCase()
+    .padStart(6, "0");
+  return `U+${hex}`;
+}
+
+function printStringCodePoints(string) {
```

Comment:
Note: I’ve extracted the `printCodePoint` and `printStringCodePoints` logic, but haven’t added these to the `defines` frontmatter above since they’re not used directly in any tests. They’re just called from the other helpers (which are in `defines`) in this file. Let me know if they need to be added to `defines` regardless.

Reply:
> frontmatter above since they’re not used directly in any tests.

That's perfect, thank you.